frontend/src/pages/Analytics.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+
+/***ASSETS***/
+import Analyze from '../assets/analyze.webp';
+import Analysis from '../assets/analysis.webp';
+
+/***COMPONENTS***/
+import NavBar from '../components/NavBar';
```

Comment:
After checking all the build logs and your commit history, it seems that you made a typo here, since the B is capitalized here but the B isn't in the filename. For the function itself, the B is capitalized but not the filename.

Reply:
> After checking all the build logs and your commit history, it seems that you made a typo here, since the B is capitalized here but the B isn't in the filename. For the function itself, the B is capitalized but not the filename.

![image](https://user-images.githubusercontent.com/67027837/204937279-9a023a02-d5f9-4054-9a11-22661e3da40d.png)
That's odd because my local branch has always had it updated as NavBar.tsx but that change didn't get pushed in the last few commits even though it was updated (you can see there are no changes to stage).

Reply:
So I just discovered that git doesn't recognize capitilization changes of file names as being changes so it wasn't staged and pushed.